lib/HLSL/DxilPatchShaderRecordBindingsShared.h

```diff
@@ -21,10 +21,15 @@
 #define FallbackLayerDispatchConstantsRegister 0
 #define FallbackLayerAccelerationStructureList 1
 
+#ifndef _WIN32
+#define __int32 int
```

Comment:
We should probably use `int32_t` and `int64_t` here since `int` and `long long` is not guaranteed to be fixed-width.

Reply:
Ah yes. I will replace the usage of `unsigned __int32` with `uint32_t` and `unsigned __int64` with `uint64_t`.

According to the reference (https://docs.microsoft.com/en-us/cpp/cpp/int8-int16-int32-int64?view=vs-2017):
> __int32 is synonymous with type int. The __int64 type is synonymous with type long long.

Also according to the reference (https://msdn.microsoft.com/en-us/library/mt764276.aspx):
> typedef unsigned int       uint32_t;
typedef unsigned long long uint64_t;